apps/site/pages/components/organisms/shipping-simulation.mdx

```diff
@@ -0,0 +1,336 @@
+---
+description: Shipping Simulation are sliders that allows users to select a maximum and minimum price.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/PriceRange___5863b0eb4a4a8ce9a98da5c760461e61.png
+---
+
+import path from 'path'
+import { Tab, Tabs } from 'nextra-theme-docs'
+import { useSSG } from 'nextra/ssg'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { ShippingSimulation } from '@faststore/ui'
+import { useFormattedPrice } from 'site/components/utilities/usePriceFormatter'
+import ShippingSimulationUsage from 'site/components/ShippingSimulation/ShippingSimulationUsage'
+import { getComponentPropsFrom } from 'site/components/utilities/propsSection'
+
+export const getStaticProps = () => {
+  const shippingSimulationPath = path.resolve(__filename)
+  const components = [
+    'ShippingSimulation.tsx',
+  ]
+  const [shippingSimulationProps] =
+    getComponentPropsFrom(shippingSimulationPath, components)
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: {
+        shippingSimulationProps
+      },
+    },
+  }
+}
+
+export const ShippingSimulationPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { shippingSimulationProps } = useSSG()
+  return {
+    ShippingSimulation: <PropsSection propsList={shippingSimulationProps} />,
+  }[component]
+}
+
+<header>
+
+# Shipping Simulation
+
+This component lets customers set the `PostalCode` and see the shipping options in the `PDP`.
+
+</header>
+
+## Overview
+
```

Comment:
If this example the same as the listed at `Usage` section? If so, we can remove this `Overview` section ;)